### DISCUSSION TITLE
canva github student pack

### DISCUSSION THREAD

**Discussion by AzizBenIsmail** [community/community#50799] [GitHub Education]
Is canva out of the pack ... ?

**Comment by samus-aran** [comment]
Hi @AzizBenIsmail  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers).
